go.mod

```diff
@@ -16,14 +15,14 @@ require (
 	github.com/siliconbrain/go-mapseqs v0.4.0
 	github.com/siliconbrain/go-seqs v0.13.0
 	github.com/stretchr/testify v1.10.0
-	go.opentelemetry.io/collector/component v0.114.0
-	go.opentelemetry.io/collector/config/configauth v0.114.0
-	go.opentelemetry.io/collector/config/configcompression v1.20.0
-	go.opentelemetry.io/collector/config/configopaque v1.20.0
-	go.opentelemetry.io/collector/config/configtelemetry v0.114.0
-	go.opentelemetry.io/collector/otelcol v0.114.0
-	go.opentelemetry.io/collector/pipeline v0.114.0
-	go.opentelemetry.io/collector/service v0.114.0
+	go.opentelemetry.io/collector/component v0.115.0
+	go.opentelemetry.io/collector/config/configauth v0.115.0
+	go.opentelemetry.io/collector/config/configcompression v1.21.0
+	go.opentelemetry.io/collector/config/configopaque v1.21.0
+	go.opentelemetry.io/collector/config/configtelemetry v0.115.0
+	go.opentelemetry.io/collector/otelcol v0.115.0
+	go.opentelemetry.io/collector/pipeline v0.115.0
+	go.opentelemetry.io/collector/service v0.115.0
```

Comment:
We _might_ have some trouble with this as the `axoflow-otel-collector` is currently at [`v0.112.0`](https://github.com/axoflow/axoflow-otel-collector-releases/releases/tag/v0.112.0-dev10).